askup/utils/general.py

```diff
@@ -369,3 +456,12 @@ def get_student_last_week_incorrect_answers_count(user_id):
         return cursor.fetchone()[0] or 0
 
     return 0
+
+
+def get_user_organizations_string(user):
+    """
+    Return comma separated user organizations string.
+    """
+    return ', '.join(
+        (str(org) for org in askup.models.Organization.objects.filter(users__in=[user]))
```

Comment:
Why do you use `__in` instead of simple equal?

Reply:
thank you. fixed. didn't know that you can compare a single object here to the query set.

Reply:
@KhaimovMR What did you fix? I see this comment and not sure I am right.

Reply:
Yeah. I've tried to compare a users set to a single user object:
`filter(users=user)`
and it worked